modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -819,15 +819,17 @@ private boolean pingNode(TcpDiscoveryNode node) {
      * @param nodeId Node ID to ping. In case when client node ID is not null this node ID is an ID of the router node.
      * @param clientNodeId Client node ID.
      * @param timeout Timeout on operation in milliseconds. If 0, a value based on {@link TcpDiscoverySpi} is used.
+     * @param debugLogging Boolean flag indicating whether debug information should be printed into node log.
```

Comment:
Javadoc is confusing

Reply:
changed to "@param debugLogging Debug flag